block/serialize.go

```diff
@@ -0,0 +1,301 @@
+package block
+
+import (
+	"github.com/vulpemventures/go-elements/internal/bufferutil"
+)
+
+func (b *Block) SerializeBlock() ([]byte, error) {
+	s, err := bufferutil.NewSerializer(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	err = b.Header.SerializeHeader(s)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
can you change to this code-style here but also elsewhere in this package:
```suggestion
	if err := b.Header.SerializeHeader(s); err != nil {
		return nil, err
	}
```

Reply:
Done